docs/en/sql-reference/aggregate-functions/index.md

```diff
@@ -72,3 +72,16 @@ FROM t_null_big
 │ 2.3333333333333335 │                 1.4 │
 └────────────────────┴─────────────────────┘
 ```
+
+Also you can use [Tuple](../data-types/tuple.md) to change NULL skipping behavior.
+
+```sql
+select groupArray(b), groupArray(tuple(b)).1 from t_null_big;
+
+┌─groupArray(b)─┬─tupleElement(groupArray(tuple(b)), 1)─┐
+│ [2,2,3]       │ [2,NULL,2,3,NULL]                     │
+└───────────────┴───────────────────────────────────────┘
```

Comment:
```suggestion
SELECT
    groupArray(y),
    groupArray(tuple(y)).1
FROM t_null_big;

┌─groupArray(y)─┬─tupleElement(groupArray(tuple(y)), 1)─┐
│ [2,2,3]       │ [2,NULL,2,3,NULL]                     │
└───────────────┴───────────────────────────────────────┘
```

The motivation is to be consistent with the formatting and use the correct column names.